server/config/topic_page/poverty/USA.textproto

```diff
@@ -18,6 +18,10 @@ metadata {
   topic_id: 'poverty'
   topic_name: 'Poverty'
   place_dcid: 'country/USA'
+  contained_place_types {
```

Comment:
were these topic page changes supposed to be part of this pr?

Reply:
it was a quick fix, but will be superseded by the more recent change #1441